doc/content/concepts/devicetemplates/_index.md

```diff
@@ -0,0 +1,13 @@
+---
+title: "Device Templates"
+description: ""
+weight: 1
+---
+
+Device templates can be used to quickly create large amounts of end devices with common settings. Templates can be created from existing devices or converted from an input file. You can create end devices directly from templates. See [Creating Templates](./creating), [Converting Templates](./converting) and [Executing Templates](./executing).
```

Comment:
Use the [`relref` shortcode](https://gohugo.io/content-management/cross-references/) for links, that gives us free ref checking. Also in other doc content.